bindings/go/mcp/mcp.go

```diff
@@ -0,0 +1,97 @@
+package main
+
+import (
+    "os"
+    "io"
+    "fmt"
+    "flag"
+    "log"
+    "../mio"
+)
+
+func usage() {
+    fmt.Fprintf(flag.CommandLine.Output(),
```

Comment:
Why not `fmt.Printf()`?

Reply:
This is taken from an example at https://golang.org/pkg/flag/#pkg-variables. Probably it's because this function is called from the `flag` pkg which can be changed with flag.SetOutput()?